extensions/wikia/FacebookClient/tests/FacebookClientTest.php

```diff
@@ -7,6 +7,15 @@ public function setUp() {
 	}
 
 	public function testGetReturnToUrl() {
+
+		// TODO: remove this once FB is turned on globally and unconditionally
```

Comment:
Ah, there's the internal wikia where we don't load this.  Still, this should be the default for all production wikia's so lets take out this check for ```wgEnableFacebookClientExt```.  I'd rather this die and tell us the test environment doesn't match production.